package.json

```diff
@@ -10,7 +10,8 @@
     "dev": "concurrently \"npx tsc -w\" \"nodemon ./src/index.ts\"",
     "migrate": "cp .env ./src/database/ && npx knex migrate:latest --knexfile src/database/knexfile.ts; rm ./src/database/.env",
     "unit-tests": "jest unit",
-    "integration-tests": "jest integration"
+    "integration-tests": "jest integration",
+    "lint": "npx eslint . --fix"
```

Comment:
Do we really want eslint to fix all problems for us or do we want it to show us all problems (without fixing) ?

Reply:
Eslint will only fix automatic fixable problems (for exemple replacing the "let" keyword by as "const" if the variable is not modified in the whole code). But for more complex problems, it will only show them to you (and you'll have to solve them by yourself) .

Reply:
Ok it makes sense. Perfect